packages/core/handlers/routers/user.js

```diff
@@ -1,20 +1,39 @@
 const express = require('express');
 const { createAppHandler } = require('../app-handler-helpers');
 const { checkRequiredParams } = require('@friggframework/core');
-const { User } = require('../backend-utils');
+const { User } = require('../../user/user');
+const { UserRepository } = require('../../user/user-repository');
+const { UserFactory } = require('../../user/user-factory');
+const {
+    CreateIndividualUser,
+} = require('../../user/use-cases/create-individual-user');
+const { LoginUser } = require('../../user/use-cases/login-user');
+const {
+    CreateTokenForUserId,
+} = require('../../user/use-cases/create-token-for-user-id');
 const catchAsyncError = require('express-async-handler');
 
 const router = express();
 
+// A poor-man's dependency injection container
```

Comment:
Ah I see. Hadn't seen this pattern before. Downsides? More verbose/scaffolding in a given file?